examples/custom-config/postgres-ha.yaml

```diff
@@ -0,0 +1,18 @@
+---
+bootstrap:
+  dcs:
+    postgresql:
+      parameters:
+        log_directory: pglogs
+        log_min_duration_statement: 0
+        log_statement: none
+        max_wal_senders: 6
+        shared_preload_libraries: pgaudit.so
```

Comment:
If we add in `pg_stat_statements` (which we do in https://github.com/CrunchyData/postgres-operator/pull/1184/commits/aaf69811b6e361ecf1b1d1b7168d19aacd9fa9a5#diff-64d4c5af3fcff84b3768e377f9654aa6R4-R5) we need to add it to `shared_preload_libraries`:

```yaml
shared_preload_libraries: "pgaudit.so,pg_stat_statements"
```

Reply:
I will go ahead and update the `setup.sql` to no longer create the `pg_stat_statements` extension.  We typically enable both of these by default, and the intention here was to simply show the ability to customize that behavior.